bot_sample.go

```diff
@@ -8,8 +8,7 @@ import (
 	"os/signal"
 	"regexp"
 	"strings"
-
-	"github.com/mattermost/platform/model"
+	"github.com/mattermost/mattermost-server/model"
```

Comment:
I suspect that this is the only necessary change to solve the problem. We split the old `mattermost/platform` repository into `mattermost/mattermost-server` and `mattermost/mattermost-webapp` awhile back, and forgot to update this sample project.

The other changes in these files are likely unnecessary, and should be reverted.

Reply:
you are right, only one of the two commits should have been in the request. i will change.
however, the change in glide.yaml should be merged too, yes?

Reply:
i created a new merge request containing only the one commit

Reply:
Ok, I'm going to close this request then. Thanks for the submission!